_2017_pages/schedule.html

```diff
@@ -4,332 +4,596 @@
+
+
+	<link rel="stylesheet" id="wp-block-library-css" href="../schedule_files/style.min.css" type="text/css" media="all">
+<link rel="stylesheet" id="bodhi-svgs-attachment-css" href="../schedule_files/svgs-attachment.css" type="text/css" media="all">
+<script type="text/javascript" src="../schedule_files/jquery.js"></script>
+<script type="text/javascript" src="../schedule_files/jquery-migrate.min.js"></script>
+<script type="text/javascript">
+/* <![CDATA[ */
+var cssTarget = "img.style-svg";
+var ForceInlineSVGActive = "false";
+
+</script>
+<script type="text/javascript" src="../schedule_files/svgs-inline-min.js"></script>
+<meta name="msapplication-TileColor" content="#da532c">
+<meta name="msapplication-config" content="/wp-content/uploads/fbrfg/browserconfig.xml">
+
+
+    <!-- Typekit fonts -->
+    <link rel="stylesheet" href="../schedule_files/gbq5zcz.css">
+
+    <!-- Stylesheets -->
+            <!-- Bootstrap -->
+      <link href="../schedule_files/bootstrap.min.css" rel="stylesheet">
+
+      <!-- Plugins -->
+      <link href="../schedule_files/animate.css" rel="stylesheet">
+      <link href="../schedule_files/slick2.css" rel="stylesheet">
+      <link href="../schedule_files/magnific-popup.css" rel="stylesheet">
+      <link href="../schedule_files/font-awesome.css" rel="stylesheet">
+      <link href="../schedule_files/streamline-icons.css" rel="stylesheet">
+
+      <!-- Our styles -->
```

Comment:
You should take advantage of the layouts and include files there are available. Take a look for example at the source code of the home page: https://github.com/pymc-devs/pymcon/blob/gh-pages/_layouts/2017_home.html or the code of conduct page: https://github.com/pymc-devs/pymcon/blob/gh-pages/_2017_pages/code_of_conduct.html. 

I'd recommend creating a schedule layout based on the [`2017_default`](https://github.com/pymc-devs/pymcon/blob/gh-pages/_layouts/2017_default.html) to split layout (css styles mostly) and content. Moreover, this layout will include head, header, js and footer which should be common in all pages (head for base style and metadata and analytics, header for the navbar, footer for copyright and contact links...). 

Including head and header is a must, splitting the content into the two layout/content files is not, I understand it takes some time and we currently don't have plans to use this layout anywhere else so the improvement is only regarding readablility and the ability for anyone to edit the schedule (i.e. Aki's Q&A)

Reply:
I don't have the bandwidth to look into mixing and matching metadata and navbar. 
This is as far as I can take the redesign